.node-version

```diff
@@ -1 +1 @@
-20.8.1
+20.9.0
```

Comment:
Per https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/299, let's hold off on the Node 20.9 upgrade until CircleCI and Docker support it

Reply:
Docker support is out, but not CircleCI yet going by the docs